File: testdata/outline/csharp/Service.cs
namespace Demo
{
    public interface IService
    {
        string Name();
    }

    public class Registry : IService
    {
        private readonly string label;

        public Registry(string label)
        {
            this.label = label;
        }

        public string Name()
        {
            return label;
        }

        public Uri PathToUri(string path)
        {
            return new Uri(path);
        }
    }

    public enum Level
    {
        Low,
        High
    }
}

File: testdata/outline/java/Service.java
package demo;

public interface Service {
    String name();
}

enum Level {
    LOW,
    HIGH;
}

record Point(int x, int y) {
}

public class Registry implements Service {
    private final String label;

    Registry(String label) {
        this.label = label;
    }

    public String name() {
        return label;
    }

    class Inner {
        int depth() {
            return 1;
        }
    }
}
